portfolio/src/main/webapp/style.css

```diff
@@ -68,56 +73,68 @@ form button{
   text-shadow: none;
 }
 
-.form-content{
+.form-content {
   align-items: flex-start;
   display: grid;
   grid-template-columns: 1fr 4fr;
   padding: 5px;
 }
 
-.navbar{
+#logInLogOut {
+  margin: 10px 5px;
+  padding: 10px;
+}
+
+#logInLogOut a {
+  text-decoration: none;
+  font-size: 1.063rem;
```

Comment:
for specific decimals like these, might be best to just round to an easier decimal such as 1.1 or even 1 rem if that works with your styling